hail/src/main/scala/is/hail/expr/types/physical/PLocus.scala

```diff
@@ -23,13 +23,11 @@ abstract class PLocus extends ComplexPType {
 
   def rg: ReferenceGenome
 
-  def contig(address: Long): Long
-
-  def contig(address: Code[Long]): Code[Long]
+  def contig(value: Long): String
```

Comment:
This remains necessary to support `RegionValueVariant`

Reply:
bleh.